__tests__/fixtures/index/run-custom-script-with-arguments/echo.js

```diff
@@ -0,0 +1 @@
+console.log(`A message from custom script with args ${process.argv[process.argv.length - 1]}`);
```

Comment:
Can you elaborate? I haven't considered that and don't understand what exactly you're proposing but it sounds interesting :)

Reply:
You could

```
console.log(JSON.stringify(process.argv))
```

Then in the test:

```
expect(JSON.parse(stdout)).toEqual([ '--help' ]);
```

I think it would be semantically more sound because we wouldn't test the rest of the message on top of the args (A message from custom script etc).

Reply:
Like it. Done!